source/includes/tutorials/stop-containers.rst

```diff
@@ -1,26 +1,42 @@
-After you complete this tutorial, you can free resources by stopping the
-tutorial Docker containers and MongoDB data. Navigate to the tutorial
-directory "mongodb-kafka-base" that you created in the setup step and run the
-following command:
+After you complete this tutorial, free resources on your computer
```

Comment:
```suggestion
After you complete this tutorial, you can free up resources on your computer
```

Reply:
I think you don't need the "up" and I think the "you can" takes away from the imperative mood of this instruction. I'll hold off on performing this edit for the time being.